### DISCUSSION TITLE
Update requires MSI of currently installed version; MSI of currently installed version flagged as virus by Chrome/FF/Edge

### DISCUSSION THREAD

**Discussion by jpers36** [gitextensions/gitextensions#9578] [Q&A]
Windows 10

Trying to install 3.5.3 over 3.4.0.  When running the MSI it asks for the location of the 3.4.0 MSI.  I don't have it, and I can't download it as every browser I have flags it as malware.  The installer apparently can't continue without it.

- Would uninstalling from Apps & Features require the old MSI as well?
- Is there a manual uninstall process if an Apps & Features uninstall fails?
- What configuration settings would I lose pursuing one of these avenues, and is there any way to mitigate?

**Comment by RussKie** [comment]
You can download previous releases from https://github.com/gitextensions/gitextensions/releases

**Reply by jpers36** [reply]
I cannot.  My browsers flag it as malware.

**Reply by RussKie** [reply]
I get the same, wow...
![image](https://user-images.githubusercontent.com/4403806/133449006-6fb1d70c-72cd-455d-b6fc-547aa8066880.png)

All your settings are stored under:
* `C:\Users\<user>\AppData\Roaming\GitExtensions\GitExtensions`
* `C:\Users\<user>\AppData\Local\GitExtensions`

You can uninstall/delete `C:\Program Files (x86)\GitExtensions` and install afresh.

**Reply by jpers36** [reply]
I started my week by addressing this.  I also needed to wipe out an Installer entry in the registry referencing the msi.  Thanks!
